### ISSUE TITLE
Granular Length Step FX

### ISSUE THREAD

**Issue by RENEGADE-ANDROiD** [polyend/TrackerBetaTesting#1548]
I have a lot of experience with the Polyend Tracker and my only request would be for the addition of a Granular Length Step FX.  It could really expand creativity with the granular engine.  Being able to adjust a length value per step would allow for some very interesting and cool sound manipulations.  I'd appreciate the consideration of this suggestion.

**Comment by NoSitRecords** [comment]
This!!

**Comment by bambrose** [comment]
Yeah would be amazing if possible

**Comment by trutypesounds** [comment]
YES

**Comment by CarloAvanti** [comment]
Really missing this feature !!

**Comment by Undistraction** [comment]
Came here to ask for the same thing. With the right settings, manually manipulating the length gives lovely results, especially moving around very low values. It would be wonderful to allow Granular length to be manipulated by FX and by LFO/envelope for that instrument.

**Comment by solipsvs** [comment]
yes please!  give it a cc too!  make it a target of modulation and fx
